""" Exception generated by config.py """

class ConfigError(Exception):
    """ Local exception class """
